drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -20,6 +20,9 @@ class Hmis::Hud::Project < Hmis::Hud::Base
   has_many :project_cocs, **hmis_relation(:ProjectID, 'ProjectCoc'), inverse_of: :project, dependent: :destroy
   has_many :inventories, **hmis_relation(:ProjectID, 'Inventory'), inverse_of: :project, dependent: :destroy
   has_many :funders, **hmis_relation(:ProjectID, 'Funder'), inverse_of: :project, dependent: :destroy
+  has_many :external_ids, class_name: 'HmisExternalApis::ExternalId', as: :source
+  has_many :external_referral_requests, class_name: 'HmisExternalApis::ReferralRequest', dependent: :destroy
+  has_many :external_referral_postings, class_name: 'HmisExternalApis::ReferralPosting', dependent: :destroy
```

Comment:
These relationships should be injected via extension from the `hmis_external_apis`